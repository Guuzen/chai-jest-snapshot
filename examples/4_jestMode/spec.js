require("./helper");
const expect = require("chai").expect;

describe("using chaiJestSnapshot in Jest mode (i.e. within Jest)", function() {
  it("delegates to Jestâ€™s built-in snapshot ability", function() {
    expect({ foo: "bar" }).to.matchSnapshot();
  });
});
