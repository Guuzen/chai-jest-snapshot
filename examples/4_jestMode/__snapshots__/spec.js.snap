// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`using chaiJestSnapshot in Jest mode (i.e. within Jest) delegates to Jestâ€™s built-in snapshot ability 1`] = `
Object {
  "foo": "bar",
}
`;
